.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature request
+about: Marked doesn't do this thing and I think it should
+
+---
+
+**Is your feature request related to a problem? Please describe.**

```

Comment:
Not feeling this set...it has some potential overlap with a bug report. If it's cool with you, once we merge and verify I might submit another PR as follow up. (To the line count comment combined it's the same count. :) )

Don't think the marked version makes sense here.

Reply:
Yeah there's room for improvement. But after reading through it a couple times, I think it makes sense. A feature request should be rooted in a problem that needs to be solved.

Maybe we can change this question to say something like "Why is this feature valuable?"

Reply:
Yes, but in your example about iTunes offering a way to purchase rented movies, you do have an implicit problem that has not been stated. Maybe the problem is that you can't watch rented movies more than once, so buying it solves that problem.

I'll change this to "why is this feature valuable" so we can at least get a Why from the person requesting it.